CGMBLEKit Example/NSUserDefaults.swift

```diff
@@ -36,4 +37,8 @@ extension UserDefaults {
             set(newValue, forKey: "transmitterID")
         }
     }
+
+    var glucoseUnit: HKUnit {
+        return HKUnit.milligramsPerDeciliter()
+    }
```

Comment:
This doesn't make sense here.

Reply:
Plan was to support mmol/L, and set the units in one place. Probably better not to complicate things for example app. Reverted.